### PR TITLE
ZEN-29254 Use separate Solr instance by BaseTestCase

### PR DESCRIPTION
**Pull request by karolzen**
https://jira.zenoss.com/browse/ZEN-29254

### CODE REVIEW

File: bin/runtests [hunk 1]
```diff
@@ -1,471 +1,37 @@
-#! /usr/bin/env python
+#!/bin/sh
```
Comment (jpeacock-zenoss) [inline bin/runtests]: To be sure, this should be `/bin/bash`.  On Ubuntu, `/bin/sh` is not `bash`.

Comment (karolzen) [reply bin/runtests]: 👍

**Comment by jpeacock-zenoss** [comment]
retest this

**Comment by jpeacock-zenoss** [comment]
retest this

**Pull request closed without merging**